src/test/java/com/amazon/opendistroforelasticsearch/security/cache/CachingTest.java

```diff
@@ -58,7 +58,7 @@ public void testRestCaching() throws Exception {
 
         Assert.assertEquals(3, DummyHTTPAuthenticator.getCount());
         Assert.assertEquals(1, DummyAuthorizer.getCount());
-        Assert.assertEquals(3, DummyAuthenticationBackend.getAuthCount());
+        Assert.assertEquals(1, DummyAuthenticationBackend.getAuthCount());
```

Comment:
Why is this change necessary? It doesn't look related to your change.

Reply:
My change fixes caching auth credentials, so the authentication backend is hit less. That's how I found this bug in the first place.